HerPortal/DataStores/UserDataStore.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading.Tasks;
```

Comment:
I'm not sure what this class is doing for us? Will it be more useful later on?

Reply:
In this case I think so. I don't think we're planning to do anything complicated with users so this class doesn't seem to be adding anything. Apologies in advance if we decide to add it back later on :)